app/services/random_phrase_service.rb

```diff
@@ -0,0 +1,14 @@
+class RandomPhrase 
+    def generate_phrase(type)
```

Comment:
Nice and simple code here!! 100% Dev empathy

Reply:
Agreed!